examples/config/cosmodc2_config.yml

```diff
@@ -16,6 +16,22 @@ TXCosmoDC2Mock:
     flip_g2: True # to match metacal
 
 
+TXMainMaps:
+    chunk_rows:  100000
+    pixelization: healpix
+    nside: 512
+    sparse: True
+
+TXAuxiliaryMaps:
+    chunk_rows:  100000
+    sparse:  True
```

Comment:
also, only the main map has the parameter "pixelization", why?

Reply:
See discussion below